packages/web-components/fast-components/package.json

```diff
@@ -28,8 +28,8 @@
     "dev": "tsc -p ./tsconfig.json -w",
     "tdd": "yarn dev & yarn test-chrome:watch",
     "prepare": "yarn clean:dist && yarn build",
-    "prettier": "prettier --config ../../../.prettierrc --write \"**/*.{ts,html}\"",
-    "prettier:diff": "prettier --config ../../../.prettierrc \"**/*.{ts,html}\" --list-different",
+    "prettier": "prettier --config ../../../.prettierrc --write \"**/*.{ts,html,md}\"",
```

Comment:
So prettier doesn't actually do anything with the markdown file, we technically only need to check it in the diff, but it seemed odd to not mirror the prettier script with the extension. I can remove though, thoughts?